openpype/hosts/max/plugins/publish/extract_pointcache.py

```diff
@@ -89,3 +85,54 @@ def process(self, instance):
             "stagingDir": parent_dir,
         }
         instance.data["representations"].append(representation)
+
+    def _set_abc_attributes(self, instance):
+        start = instance.data["frameStartHandle"]
+        end = instance.data["frameEndHandle"]
+        attr_values = self.get_attr_values_from_data(instance.data)
+        custom_attrs = attr_values.get("custom_attrs", False)
+        rt.AlembicExport.ArchiveType = rt.Name("ogawa")
+        rt.AlembicExport.CoordinateSystem = rt.Name("maya")
+        rt.AlembicExport.StartFrame = start
+        rt.AlembicExport.EndFrame = end
+        rt.AlembicExport.CustomAttributes = custom_attrs
+
+    @classmethod
+    def get_attribute_defs(cls):
+        return [
+            BoolDef("custom_attrs",
+                    label="Custom Attributes",
+                    default=False),
+        ]
+
+
+class ExtractCameraAlembic(ExtractAlembic):
+    """Extract Camera with AlembicExport."""
+
+    order = pyblish.api.ExtractorOrder - 0.1
+    label = "Extract Alembic Camera"
+    hosts = ["max"]
+    families = ["camera"]
+    optional = True
+
+
+class ExtractModel(ExtractAlembic):
+    """
+    Extract Geometry in Alembic Format
+    """
+
+    order = pyblish.api.ExtractorOrder - 0.1
+    label = "Extract Geometry (Alembic)"
+    hosts = ["max"]
+    families = ["model"]
+    optional = True
+
+    def _set_abc_attributes(self, instance):
+        attr_values = self.get_attr_values_from_data(instance.data)
+        custom_attrs = attr_values.get("custom_attrs", False)
+        rt.AlembicExport.ArchiveType = rt.name("ogawa")
+        rt.AlembicExport.CoordinateSystem = rt.name("maya")
+        rt.AlembicExport.CustomAttributes = custom_attrs
+        rt.AlembicExport.UVs = True
+        rt.AlembicExport.VertexColors = True
+        rt.AlembicExport.PreserveInstances = True
```

Comment:
Is there a reason these need to be extracted earlier?

We should only need to override what we actually want to differ from the `ExtractAlembic` parent class.

```suggestion
class ExtractCameraAlembic(ExtractAlembic):
    """Extract Camera with AlembicExport."""
    
    label = "Extract Alembic Camera"
    families = ["camera"]


class ExtractModel(ExtractAlembic):
    """Extract Geometry in Alembic Format"""

    label = "Extract Geometry (Alembic)"
    families = ["model"]

    def _set_abc_attributes(self, instance):
        attr_values = self.get_attr_values_from_data(instance.data)
        custom_attrs = attr_values.get("custom_attrs", False)
        rt.AlembicExport.ArchiveType = rt.name("ogawa")
        rt.AlembicExport.CoordinateSystem = rt.name("maya")
        rt.AlembicExport.CustomAttributes = custom_attrs
        rt.AlembicExport.UVs = True
        rt.AlembicExport.VertexColors = True
        rt.AlembicExport.PreserveInstances = True
```